docs/pages/Configuration.md

```diff
@@ -303,6 +317,20 @@ builder:
         color: blue
 ````
 
+### JSDoc
+You can exclude a projects resources from the JSDoc build process using a list of glob patterns. Matching resources will be ignored by the JSDoc build task.
```

Comment:
You can exclude the resources of a project ...

Reply:
See https://github.com/SAP/ui5-tooling/pull/258